docs/install/custom-server.rst

```diff
@@ -31,11 +31,25 @@ Step 1: Installing The Littlest JupyterHub
 #. Using a terminal program, SSH into your server. This should give you a prompt where you can
    type commands.
 
+#. If your server is behind a firewall and needs a proxy:
+
+   .. code-block:: bash
+
+      export http_proxy=<your_proxy>
+      export https_proxy=<your_proxy>
+
+#. Some requests will fail if your certs are self-signed:
```

Comment:
Do you have a sense of why this is? Fiddling with CA stores is always dangerous, and I do not want us to do that unless we explicitly need it.

Reply:
I *think* this is because your organization intercepts HTTPS with some sort of MITM to inspect all traffic in and out. Do you know if that is the reason? Can you tell me what error you got and who the 'self' in self-signed is?